.github/workflows/windows.yml

```diff
@@ -37,9 +37,9 @@ jobs:
         mkdir build
         cd build
         cmake .. -A x64
-        cmake --build . --config Release
+        cmake --build . -j --config Release
```

Comment:
Isn't it necessary to write the number of threads after the parameter `j`?

Reply:
The man page said it would default to the build tools default, but I forgot make would just become unbounded. I've added a parameter.